test/metabase/api/card_test.clj

```diff
@@ -680,7 +680,7 @@
 
 ;; Test that we *cannot* share a Card if the Card has been archived
 (expect
-  "The object has been archived."
+  {:message "The object has been archived.", :error_code "archived"}
```

Comment:
@tlrobinson I added that after talking with @attekei about it. There're no HTTP status codes that seemed appropriate and we wanted a way to differentiate different kinds of 404s so this is what we came up with.